session-1-jigsaw-intro/07_patch_module_option/compile.sh

```diff
@@ -31,5 +32,18 @@ javac --patch-module java.base=src \
       src/java.base/java/util/concurrent/ConcurrentHashMap.java
 
 echo ""
+echo "${info} *** Compiling Main class*** ${normal}"
```

Comment:
Im not sure if this is necessary for the patching to work, this module should not/is not used.